content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,270 @@
+
+Depending on which AWS services you're using and your use case for monitoring, there are multiple options within the integration tile to specify the data to be collected. For example, you can limit data collection based on AWS service, namespace, or tags. Additionally, you can choose to mute monitor notifications, for instance terminations triggered manually or by autoscaling with [EC2 automuting][14] enabled. If needed, enable [Alarm Collection][15] to send your CloudWatch alarms to the Datadog [Event Stream][16] and choose whether to collect custom metrics.
+
+### Validation
+
+View the out-of-the-box [AWS overview dashboard][17] to see metrics sent by your AWS services and infrastructure:
+{{< img src="getting_started/integrations/aws-dashboard.png" alt="An image displaying the AWS overview dashboard in the Datadog account. On the left is the AWS logo and an AWS events graph showing 'No matching entries found'. In the center are graphs related to EBS volumes with numerical data displayed and a heat map showing consistent data. Along the right are graphs related to ELBs showing numerical data as well as a timeseries graph showing spikey data from three sources.">}}
+
+## Enable sub-integrations
+
+See the [Integrations page][18] for a full listing of the available sub-integrations.
+
+## Send logs
+
+### With CloudTrail
+
+As [CloudTrail][19] is widely used and helps to power [Cloud SIEM][20], follow the [Log collection documentation][21] for CloudTrail. This requires sending your CloudTrail logs to S3 and adding a trigger on the already-provisioned Datadog Forwarder Lambda function for `Object Created (All)` events on your S3 bucket.
+
+Any new log events will be sent to Datadog once established, and the logs appear with the `Source` and `Service` attributes set as `cloudtrail`:
+
+{{< img src="getting_started/integrations/logs-explorer-cloudtrail.png" alt="An image from the Logs Explorer page of the Datadog account. Along the left the image displays the Source facet, checked with 'cloudtrail'. Along the right, some log entries are displayed in a list format.">}}
+
+### Automatically using triggers
```

Comment:
Remove this section. I think it does more harm than good in this context since it doesn't work easily without a lot of hidden steps.